jenkinsci/tfs-plugin
# Clarify the plugin's SCM is for Team Foundation Version Control (TFVC)

One way to prevent JENKINS-33280 is to better document what's expected, since the documentation had fallen behind the implementation details.  These changes provide better examples and explanations of the SCM fields.

Manual testing
--------------

Visit a job configured to use the `TeamFoundationServerScm` and notice only the labels, documentation, etc. have changed and the rest works the same:

![image](https://cloud.githubusercontent.com/assets/297515/17156260/de8edb02-5356-11e6-87dd-1a423a07c011.png)

Mission accomplished!
